bookstore-rust/entity/src/book.rs

```diff
@@ -0,0 +1,19 @@
+use chrono::NaiveDate;
+use sea_orm::entity::prelude::*;
+use serde::{Deserialize, Serialize};
+
+#[derive(Clone, Debug, PartialEq, Eq, DeriveEntityModel, Deserialize, Serialize)]
+#[sea_orm(table_name = "books")]
+pub struct Model {
```

Comment:
This is not a model but an entity... so ideally, you should have a model in between and it should not be called Model (it's also in the entity package)

Reply:
It has to be named "Model" that is a Sea-Ort requirement. 🤷🏼‍♂️

Reply:
What if you have multiple entities? Would they all be called Model?

Reply:
I guess ... yes ... HAHA

I had it named differently in the beginning but it didn't work and all examples used "Model". I need more time to find out. 

I basically rushed through the stuff to get it to work and I have quite a couple of "blind spots" about the details.

Anyway ... the performance of this in the benchmarks will make you all go "Awwww" :)